src/genie/libs/parser/iosxe/tests/ShowWlanSummary/cli/equal/golden_output_output.txt

```diff
@@ -0,0 +1,10 @@
+
+Number of WLANs: 4
+
+ID   Profile Name                     SSID                             Status Security
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+17   lizzard_Global                  lizzard                         UP     [WPA2][802.1x][FT + 802.1x][AES],[FT Enabled]
```

Comment:
This spacing seems off, can you confirm it is correct?

Reply:
@itdependsnetworks I confirm, all examples in this issue are identical. 
[networktocode issue# 26](https://github.com/networktocode/genieparser/issues/51)